converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Configurations/PatientConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.DicomCast.Core.Configurations;
+
+public class PatientConfiguration
```

Comment:
I know we have historically called these "configurations," but we should eventually call these "options" instead